src/coralogix/schema.js

```diff
@@ -54,71 +55,79 @@ const schema = {
         end_msec: vcl`time.end.msec`,
         elapsed: '%D',
       },
-      client: {
-        name: str(vcl`client.as.name`),
-        number: vcl`client.as.number`,
-        location_geopoint: {
-          lat: vcl`client.geo.latitude`,
-          lon: vcl`client.geo.longitude`,
-        },
-        city_name: str(vcl`client.geo.city.ascii`),
-        country_name: str(vcl`client.geo.country_name.ascii`),
-        connection_speed: str(vcl`client.geo.conn_speed`),
-        ip: str(
-          vcl`regsuball(req.http.x-forwarded-for, ",.*", "")`,
-        ),
-        client_ip: str(vcl`client.ip`),
+      is_edge: vcl`if(fastly.ff.visits_this_service == 0, "true", "false")`,
+      visits_this_service: vcl`fastly.ff.visits_this_service`,
+      fastly_ff: req`Fastly-FF`,
+      datacenter: str(vcl`server.datacenter`),
+      hostname: str(vcl`server.hostname`),
+      ip: str('%A'),
+      region_code: str(vcl`server.region`),
+      is_cacheable: vcl`if(fastly_info.state ~"^(HIT|MISS)(?:-|$)", "true", "false")`,
+      cache_status: str(vcl`fastly_info.state`),
+      is_h2: vcl`if(fastly_info.is_h2, "true", "false")`,
+      is_h2_push: vcl`if(fastly_info.h2.is_push, "true", "false")`,
+      fastly_error: str(vcl`fastly.error`),
+    },
+    client: {
+      name: str(vcl`client.as.name`),
+      number: vcl`client.as.number`,
+      location_geopoint: {
+        lat: vcl`client.geo.latitude`,
+        lon: vcl`client.geo.longitude`,
       },
-      request: {
-        id: str(vcl`if(req.http.X-CDN-Request-ID, req.http.X-CDN-Request-ID, randomstr(8, "0123456789abcdef") + "-" + randomstr(4, "0123456789abcdef") + "-" + randomstr(4, "0123456789abcdef") + "-" + randomstr(1, "89ab") + randomstr(3, "0123456789abcdef") + "-" + randomstr(12, "0123456789abcdef"))`),
-        method: str('%m'),
-        protocol: str(vcl`if(fastly_info.is_h2, "HTTP/2", "HTTP/1.1")`),
-        h2: vcl`if(fastly_info.is_h2, "true", "false")`,
-        h2_push: vcl`if(fastly_info.h2.is_push, "true", "false")`,
-        is_ipv6: vcl`if(req.is_ipv6, "true", "false")`,
-        url: str(vcl`cstr_escape(if(req.http.X-Orig-Url, req.http.X-Orig-Url, req.url))`),
+      city_name: str(vcl`client.geo.city.ascii`),
+      country_name: str(vcl`client.geo.country_name.ascii`),
+      connection_speed: str(vcl`client.geo.conn_speed`),
```

Comment:
I don't think this is useful, we can as well remove it

Reply:
good point , done